### DISCUSSION TITLE
Include overlays by 1playerinsertcoin

### DISCUSSION THREAD

**Discussion by ianrobrien** [OnionUI/Onion#1414] [Ideas]
There are some great overlays by 1playerinsertcoin. I believe "Perfect CRT" has already been integrated into Onion, but there are GG, GB and GBC overlays as well.

https://www.reddit.com/user/1playerinsertcoin/

**Comment by ourigen** [comment]
Onion v4.3 now have Perfect GB, GBC, GBA , and CRT-240p overlays. See #1436 
I haven't added Perfect_GG though
